sql/index/pilosa/driver.go

```diff
@@ -79,6 +79,15 @@ type (
 func NewDriver(root string) *Driver {
 	h := pilosa.NewHolder()
 	h.Path = filepath.Join(root, "."+DriverID)
+
+	if err := h.Open(); err != nil {
+		panic(err)
```

Comment:
maybe should open the holder on LoadAll instead? to be able to return an error if necessary instead of a panic.

Reply:
It would make sense, but because we already open/close index in lookups, if LoadAll is not the first operation (but Create, Save, LoadAll) then it will panic on open field.